prov/efa/src/rdm/efa_rdm_pke_cmd.c

```diff
@@ -739,8 +739,15 @@ fi_addr_t efa_rdm_pke_insert_addr(struct efa_rdm_pke *pkt_entry, void *raw_addr)
 
 	assert(base_hdr->type >= EFA_RDM_REQ_PKT_BEGIN);
 
+	/*
+	 * The message is from a peer through efa device, which means peer is not local
+	 * or shm is disabled for transmission.
+	 * We shouldn't insert shm av anyway in this case.
+	 * Also, calling fi_av_insert internally inside progress engine is violating
+	 * Libfabric standard for FI_AV_TABLE.
+	 */
 	ret = efa_av_insert_one(ep->base_ep.av, (struct efa_ep_addr *)raw_addr,
-	                        &rdm_addr, 0, NULL);
+	                        &rdm_addr, 0, NULL, false);
```

Comment:
Was this open called by progress engine?

Reply:
`efa_rdm_pke_insert_addr` is only called inside progress engine